spec_insert/.rubocop.yml

```diff
@@ -0,0 +1,25 @@
+require: rubocop-rake
+AllCops:
+  Include:
```

Comment:
Should apply to specs to, use `rubocop -a ; rubocop --auto-gen-config` in general.